packages/cli/src/commands/init.ts

```diff
@@ -245,8 +244,6 @@ export default class InitCommand extends Command {
           subgraphName,
           contractName,
           node,
-          studio,
```

Comment:
Is this already removing some arguments from `init`?

Reply:
no we are not removing any args. This is simply removing the validation check since we only do that check during `init` fro `studio`. Should not change any args.